### PR TITLE
Add Atlas V & Atlas III RD-180 from BDB

### PR DESCRIPTION
**Pull request by vader111**
Add Atlas V & Atlas III RD-180 from BDB

### CODE REVIEW

**Comment by vader111** [comment]
(Ready To Review)

**Comment by Capkirk123** [comment]
Surface attaching is disabled for the RD-180 (it should be enabled). Also, I would recommend at least disabling the old RD-180, if not outright replacing it.

**Comment by vader111** [comment]
@Capkirk123 Fixed

**Comment by Capkirk123** [comment]
Looks good. I might suggest enabling the nozzle internal glow however.

**Comment by vader111** [comment]
@Capkirk123 Should be Fixed now

**Comment by Capkirk123** [comment]
Everything looks good

File: GameData/ROEngines/PartConfigs/RD180_BDB.cfg [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/RD180_BDB.cfg:17]: Can you make these parts [and your other PRs] use the tool in https://github.com/KSP-RO/ROEngines/blob/PatchingFixes/GameData/ROEngines/ROEEngineDefaults.cfg?

File: GameData/ROEngines/PartConfigs/RD180_BDB.cfg [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+	tags = rd180, rd-180, 180, atlas-v, atlas, v, atlas-5, 5
+
+	engineType = RD180
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+	childStageOffset = 1
+
+	TechRequired = stagedCombustion1998
+	cost = 2700
+	entryCost = 54000
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/RD180_BDB.cfg:47]: Can you make these parts [and your other PRs] use the tool in https://github.com/KSP-RO/ROEngines/blob/PatchingFixes/GameData/ROEngines/ROEEngineDefaults.cfg?

**Review by DRVeyl — CHANGES_REQUESTED** [review]

**Review by DRVeyl — APPROVED** [review]

**Comment by DRVeyl** [comment]
Can you resolve the conflict?  (Probably a rename or deletion?)

**Pull request merged**